scripts/mobile/android_jar_reduce_size.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
Small script which:
- unpack JAR
- replace all pixels in PNGs with balck color
- pack JAR again